lib/sisimai/bite/json.rb

```diff
@@ -0,0 +1,38 @@
+module Sisimai
+  module Bite
+    module JSON
+      # Sisimai::Bite::JSON - Base class for Sisimai::Bite::JSON::*
+      class << self
+        # Imported from p5-Sisimail/lib/Sisimai/Bite/JSON.pm
+        require 'sisimai/bite'
+        require 'sisimai/rfc5322'
+
+        def headerlist; return []; end
```

Comment:
__[rubocop]__
* Style/SingleLineMethods: Avoid single-line method definitions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-single-line-methods)
* Style/Semicolon: Do not use semicolons to terminate expressions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-semicolon)
* Layout/TrailingWhitespace: Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

<!-- Comment by [SideCI](https://sideci.com) -->